libs/scully/src/lib/pluginManagement/pluginConfig.ts

```diff
@@ -115,3 +115,9 @@ export const routePluginConfig = (
     plugin[configData] = plugin[backupData];
   };
 };
+
+export const setPluginPriotity = (name: string | symbol, priority: number) => {
+  const plugin = findPlugin(name);
+
+  plugin[routeProcessPriority] = priority;
+};
```

Comment:
This is most of it, but when the plugin isn't found, it should log an error.
perhaps even throw? (That is what it does now, but with an error that's meaningless to the user.)
The error should be as explicit as something.
```
You tried to set plugin priority on ${name}. This plugin isn't found.
```
I'm in doubt about throwing instead of logging, is this "critical" enough to exit the run? I would lean to yes on that, as it _is_ an error in the config file. Its just not too servere

Reply:
Thanks for pointing that out. I also think it should be an error. I added a custom error message. I tried to keep it similar to the default one but I'm happy to change it to whatever you think is best.

Since the process gets killed now in case of an error, I have no idea how I would test this. Are there any existing tests for the `findPlugin()` function I could take a look at.

Reply:
I'm Happy with this. it prevents devs from making a silly mistake. and hard exit is ok in my eye's it makes more clear _where_ the problem is.